src/hotspot/share/runtime/globals.hpp

```diff
@@ -1824,6 +1824,10 @@ define_pd_global(uint64_t,MaxRAM,                    1ULL*G);
           "class pointers are used")                                        \
           range(1*M, 3*G)                                                   \
                                                                             \
+  diagnostic(bool, PrintMetaspaceStatisticsAtExit, false,                   \
+                  "Print metaspace statistics upon VM exit.")               \
```

Comment:
Could you fix the indentation of this line to match the formatting for the other flags around here?